.github/workflows/java-sample-app-ecr-deploy.yml

```diff
@@ -27,31 +31,29 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: ${{ secrets.E2E_IAD_TEST_ACCOUNT_ARN }}
+          role-to-assume: arn:aws:iam::${{ env.E2E_TEST_ACCOUNT_ID }}:role/${{ env.E2E_TEST_ROLE_NAME }}
           aws-region: us-east-1
 
       - name: Retrieve account
         uses: aws-actions/aws-secretsmanager-get-secrets@v1
         with:
-          secret-ids:
+          secret-ids: |
             ACCOUNT_ID, region-account/${{ matrix.aws-region }}
+            JAVA_MAIN_SAMPLE_APP_IMAGE, e2e-test/java-main-sample-app-image
 
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/${{ secrets.E2E_TEST_ROLE_ARN }}
+          role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/${{ env.E2E_TEST_ROLE_NAME }}
           aws-region: ${{ matrix.aws-region }}
 
-      - name: Update ECR URI
-        working-directory: sample-apps/springboot
+      - name: Build and Upload Main Service Image v11
+        working-directory: sample-apps/java/8/springboot-main-service
         run: |
-          sed -i 's#"<ECR_IMAGE_LINK>:<TAG>"#"${{ env.ACCOUNT_ID }}.dkr.ecr.${{ matrix.aws-region }}.amazonaws.com/${{ secrets.APP_SIGNALS_E2E_FE_SA_IMG }}:latest"#g' build.gradle.kts
-
-      - name: Upload Main Service Image
-        working-directory: sample-apps/springboot
-        run: gradle jib
+          sed -i 's#"<ECR_IMAGE_LINK>:<TAG>"#"${{ env.E2E_TEST_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com/${{ env.JAVA_MAIN_SAMPLE_APP_IMAGE }}:v11"#g' build.gradle.kts
```

Comment:
Change wherever it says `<ECR_IMAGE_LINK>:<TAG>` to something like `{{ECR_IMAGE_URI}}` to make it clear it's being used for substitution

Put a comment there too if you can, not sure if `build.gradle.kts` files can be commented in

Do so everywhere

Reply:
nit: Do you need the `"` in the sed command? Should just replace what's inside, not the quotes too

Not blocking though

Reply:
Done